package.json

```diff
@@ -18,7 +18,7 @@
     "npm": ">=3.0.0"
   },
   "scripts": {
-    "lint": "aegir-lint",
+    "lint": "eslint -c node_modules/aegir/config/eslintrc.yml src test",
```

Comment:
- Q. Why did it pass with `eslint -c node_modules/aegir/config/eslintrc.yml src test`?
- A. It did not lint `examples`
- Q. Why does aegir fail silently?
- A. An eslint plugin being used in the examples folder: https://github.com/ipfs/js-ipfs/blob/master/examples/bundle-webpack/.eslintrc#L9 was missing, which resulted in an exit from `eslint`. Running `eslint` on the command line will print a custom error message for this, which does not happen when using gulp-eslint.
- Q. Why does linting still fail?
- A. Because the examples were never properly fixed.